libwallet/src/vault.rs

```diff
@@ -1,7 +1,7 @@
 //! Collection of supported Vault backends
 #[cfg(feature = "vault_os")]
 mod os;
-#[cfg(feature = "vault_pass")]
+// #[cfg(feature = "vault_pass")]
```

Comment:
?

Reply:
Side note: we need CI to test the library compiles with/without all features in different target architectures. It's easy to break compilation for no_std targets